tests/go/GofmtBearTest.py

```diff
@@ -5,3 +5,11 @@
     GofmtBear,
```

Comment:
By that assignment above, the `GofmtBear` is replaced with the `unittest.TestCase`-derived `LocalBearTest` instance returned from `verify_local_bear`. That causes further issues. Better name it `GofmtBearTest = ...`